Makefile

```diff
@@ -45,19 +45,26 @@ test: build
 			bash -c "cd /home/dockeruser/binaris && npm run test"
 
 .PHONY: publish
-publish: build require-npm-creds
+publish: build require-npm-creds require-npm-tag
+		version=$(shell cat package.json | jq -r ".version")
+		git tag $(version)
```

Comment:
shouldn't NPM_TAG be here too?
otherwise you assume single npm tag per version and the mapping is not 1 to 1

Reply:
We've never discussed the mapping not being 1:1. We use `NPM_TAG` in the target you've commented on.

Reply:
Oh I see you mean instead of the `version`. I guess that makes sense, this is just the way we do it elsewhere.